site/gatsby-site/src/components/submissions/SubmissionWizard.js

```diff
@@ -9,6 +9,10 @@ import StepThree from '../forms/SubmissionWizard/StepThree';
 const SubmissionWizard = ({ submitForm, initialValues, urlFromQueryString }) => {
   const [data, setData] = useState(initialValues);
 
+  useEffect(() => {
```

Comment:
I had to delete this before because it triggers a render when the component loads, not when the form is submitted. 
Can you make sure the form is emptied only when it is submitted?